app/assets/javascripts/discourse/app/components/topic-timeline/container.hbs

```diff
@@ -96,11 +96,11 @@
         onClick={{this.updatePercentage}}
       ></div>
 
-      {{#if this.hasBackPosition}}
+      {{#if (and this.hasBackPosition this.showButton)}}
         <div class="timeline-last-read" style={{this.lastReadStyle}}>
           {{d-icon "minus" class="progress"}}
           {{#if this.showButton}}
-            <TopicTimeline::BackButton @onClick={{this.goBack}} />
+            <TopicTimeline::BackButton @goBack={{this.goBack}} />
```

Comment:
How about renaming that arg to `onGoBack` or similar? i.e. to indicate it's a function/action/callback

Reply:
Added in last commit 👍